Tetrahedral_remeshing/include/CGAL/Tetrahedral_remeshing/internal/tetrahedral_remeshing_helpers.h

```diff
@@ -319,34 +319,101 @@ Dihedral_angle_cosine cos_dihedral_angle(const typename Gt::Point_3& i,
     return Dihedral_angle_cosine(CGAL::sign(num), CGAL::square(num), sqden);
 }
 
+// cosine of dihedral angle, computed from pre-computed and non-zero normals
+template<typename Gt>
+Dihedral_angle_cosine cos_dihedral_angle(const typename Gt::Vector_3& n1,
```

Comment:
`Dihedral_angle_cosine` might be useful for other tet-related packages. Do you plan to make it more official one way or another?

Reply:
I agree it could be useful elsewhere, for example in `perturb_mesh_3()` and `exude_mesh_3()`.
I had not planned to document it but we should do it once this PR is merged